server/views/pages/sex.njk

```diff
@@ -17,6 +19,12 @@
             <span class="govuk-caption-xl"
                   data-qa="prisoner-name">Confirm arrival of {{ data.firstName }} {{ data.lastName }}</span>
 
+            {% if data.gender === 'TRANS' %}
```

Comment:
is the `if` needed? this page will only show for trans?

Reply:
Suppose if we get a null value or another gender becomes available later down the line protects us against that confusion?